src/server/coder/workspaces.ts

```diff
@@ -92,28 +115,51 @@ async function getOrCreateCoderWorkspace(studyId: string): Promise<CoderWorkspac
         return workspaceData
     } catch (error) {
         if (error instanceof CoderApiError && (error.status === 404 || error.status === 400)) {
-            return await createCoderWorkspace(studyId, user.username)
+            return await createCoderWorkspace({
+                studyId,
+                username: user.username,
+                language: latestJob.language,
+                envVars: baseImage?.envVars,
+            })
         }
         throw error
     }
 }
 
-async function createCoderWorkspace(studyId: string, username: string): Promise<CoderWorkspace> {
+interface CreateCoderWorkspaceOptions {
+    studyId: string
+    username: string
+    language: Language
+    envVars?: Record<string, string>
+}
+
+async function createCoderWorkspace(options: CreateCoderWorkspaceOptions): Promise<CoderWorkspace> {
+    const { studyId, username, language, envVars = {} } = options
     const workspaceName = generateWorkspaceName(studyId)
 
     // Populate code files prior to launching workspace
     await initializeWorkspaceCodeFiles(studyId)
 
+    const richParameterValues: Array<{ name: string; value: string }> = [
+        {
```

Comment:
Do we want to go ahead and include passing a "real" value for the container image? Right now it's always deferring to the [default](https://github.com/safeinsights/iac/blob/9ede4810d9532e5a0962e1066be38c5c4e0c0bcf/management-app/coder-templates/aws-fargate/main.tf#L50) 🤷‍♂️ .

Reply:
sure, what should we pass?